style.css

```diff
@@ -0,0 +1,87 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+:root {
+  --neutrals: #172b4d;
+  --neutral-n400: #505f79;
+  --primary: #396df2;
+  --primary-b400: #6070ff;
+  --primary-b300: #7f8cff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-size: 16px;
+  color: var(--neutrals);
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  height: 3rem;
+  padding: 0.25rem 1.5rem;
+}
+
+header .logo {
+  font-size: 1.125rem;
+  font-weight: 700;
+  color: var(--primary-b400);
+  padding: 0.625rem 0.75px;
+}
+
+header .menu-toggle {
+  width: 2.5rem;
+  height: 2.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
+
+#hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.75rem;
+  background: url(./assets/img/header-shapes\ mobile.png) 50% 50% no-repeat;
+  min-height: 75vh;
+  background-size: 100%;
+  padding: 1rem;
+}
```

Comment:
- [ ]  The hero section should have margins as defined in the design. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hamburger menu closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 😸 .

### Current mobile menu

<div align="center" >
<img src="https://user-images.githubusercontent.com/80895497/148230517-5febb3db-e747-4bb6-acdf-1b083f7df01c.PNG" alt="img" />
</div>

 ### Original design

<div align="center">
<img src="https://user-images.githubusercontent.com/80895497/148230579-fb4ba047-b532-4bc5-ad0e-1c7147269d4c.PNG" alt="img" />
</div>

Reply:
Thanks for pointing out this bug. As far as I see, there is no margin between the two section in the design. But I did a mistake of positioning the background image at the center (horizontally and vertically. The rationale behind that decision was to make sure text remains at the middle of the image but I didn't notice it might have conflict with header. now I fixed it to top.

I made few more tweaks to match better with Figma design.

Reply:
@ahangarha thank you for your communication you did a good job on this project
Keep going 👍🏼